### PR TITLE
Fixed error Maximum allowed thread group count is 65535 when resolution is very high. 

### PR DESCRIPTION
**Pull request by FrancescoC-unity**
No fogbugz available around, but was reported in slack and forum 

**What did I test**: To repro I made my game view resolution 8K and opened a new map and notice the error. Done the same and didn't repro anymore.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/ClearLightLists.compute [hunk 1]
```diff
@@ -1,7 +1,10 @@
 #pragma kernel ClearList
 
 RWStructuredBuffer<uint>  _LightListToClear;
-int _LightListEntries;
+int2 _LightListEntriesAndOffset;
+
+#define _LightListEntries (uint)_LightListEntriesAndOffset.x
+#define _LightListOffset  (uint)_LightListEntriesAndOffset.y
 
 [numthreads(64, 1, 1)]
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/ClearLightLists.compute:9]: you do'nt used _LightListOffset   in the code, don't think it is expected

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs [hunk 2]
```diff
@@ -3059,10 +3059,28 @@ BuildGPULightListResources PrepareBuildGPULightListResources(TileAndClusterData
...
             cmd.SetComputeBufferParam(parameters.clearLightListCS, parameters.clearLightListKernel, HDShaderIDs._LightListToClear, bufferToClear);
-            cmd.SetComputeIntParam(parameters.clearLightListCS, HDShaderIDs._LightListEntries, bufferToClear.count);
+            Vector2 countAndOffset = new Vector2Int(bufferToClear.count, 0);
 
             int groupSize = 64;
-            cmd.DispatchCompute(parameters.clearLightListCS, parameters.clearLightListKernel, (bufferToClear.count + groupSize - 1) / groupSize, 1, 1);
+            int totalNumberOfGroupsNeeded = (bufferToClear.count + groupSize - 1) / groupSize;
+
+            const int maxAllowedGroups = 65535;
+            int numberOfNeededDispatches = HDUtils.DivRoundUp(totalNumberOfGroupsNeeded, maxAllowedGroups);
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs:3068]: numberOfNeededDispatches  isn't used

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/ClearLightLists.compute (hunk 1 above)
Comment (FrancescoC-unity) [reply com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/ClearLightLists.compute:9]: Oh shoot, forgot to stage the full file! Staging now and pushing

**Comment by TomasKiniulis** [comment]
Did a quick check a while ago, can confirm that no exceptions occur with very high resolutions, while I could repro them before

**Comment by sebastienlagarde** [comment]
https://github.com/Unity-Technologies/Graphics/pull/2623

**Pull request merged**